src/components/MovieCard/MovieCard.scss

```diff
@@ -23,11 +27,11 @@
   }
 }
 
-button {
+.characters-btn {
   color: $accent-color;
   background-color: $header-background;
   border: 1px solid $accent-color;
-  margin: 100px 10px 0 10px;
+  margin: 15px;
```

Comment:
When we finalize styling will we want to make this margin a % or did updating it to only 15px correct the display issue we had seen?

Reply:
15px was the fix but I agree we should probably change them later to be more responsive.